### ISSUE TITLE
Doesn't work well in pull to refresh kind of view.

### ISSUE THREAD

**Issue by ManojMM026** [PonnamKarthik/reorderableitemsview#1]
Lib looks great, But it doesn't work/scroll smooth in pull to refresh like scenario. If you nest it with any other scroll behaviour scroll starts lagging.

Also can we just have it for flutter grid view, If I am not interested in making staggered grid?

**Comment by ManojMM026** [comment]
Lib also gives me multiple global key error when I do setState as suggested in sample.

**Comment by ponnamkarthik** [comment]
set `longPressToDrag: true,` which will allow you freely perform pull to refresh

**Comment by ManojMM026** [comment]
no, that did not work. I downloaded class and changed scrollPhysics it worked. But then setState started giving error. Libs looks promising though
